telemetry/src/main/java/com/newrelic/telemetry/TelemetryClient.java

```diff
@@ -55,6 +60,26 @@ public TelemetryClient(
     this.metricBatchSender = metricBatchSender;
     this.spanBatchSender = spanBatchSender;
     this.eventBatchSender = eventBatchSender;
+    this.shutdownSeconds = 3L;
```

Comment:
I'm ok with overloading the constructor, but when doing so I believe that constructors should be chained and only have one that actually does work.  That is, this should call the other constructor and pass in the defaulted `shutdownSeconds`.  So, with that, I would change this constructor to:

```
this(metricBatchSender, spanBatchSender, eventBatchSender, DEFAULT_SHUTDOWN_SECONDS)
```

and promote 3L to a constant value.